### DISCUSSION TITLE
Student Developer Pack approved but no benefit

### DISCUSSION THREAD

**Discussion by madhan-g-p** [community/community#75912] [GitHub Education] [answered]
# Topic 
### Hi everyone,

I've recieved the confirmation from the student benefits pack application page [github student developer pack application](https://education.github.com/discount_requests/application)

![image](https://github.com/community/community/assets/104260867/9405ed64-6739-4e16-9cf0-37be546698c4)

but still the benefits are not available for me stating that
![image](https://github.com/community/community/assets/104260867/02e7b0a9-e793-46e8-bd7f-cb717d0114bf)
it's been 3 days and still the benefits are not available and no mail from the github team as well.
I have

- [x] uploaded necessary proofs
- [x] 2fa enabled
- [x] address provided in billing information

looking for any other steps do i have to do?
@community

**Comment by Abhay2201-Av** [comment]
Hi @madhan-g-p how to get it?

**Reply by madhan-g-p** [reply]
Hi @Abhay2201-Av 
you mean the student developer pack ?
you can get it via this link https://education.github.com/pack
on a condition one should be enrolled in any github approved institution and should add campus email in your github account

just make sure you completed the above three steps as mentioned in the discussion

**Comment by madhan-g-p** [comment, marked as answer]
An Update , it seems it takes maximum of  3 days from the approved day to get your benefits.
Also some people having the same issue stated that opening a disucussion on this community page , makes them to resolve it quickly , that's why created this discussion..
